frontend/packages/console-app/src/components/tour/steps/StepBadge.scss

```diff
@@ -0,0 +1,4 @@
+.co-step-badge {
+  font-size: 14px;
```

Comment:
Using values instead of PF variable because
For modal it works fine for the header `--pf-global--FontSize--md`(16px) but for popover the same variable is of 14px. And for content `--pf-global--FontSize--sm` is (13px) but needed 14px
<img width="1527" alt="Screenshot 2020-07-30 at 11 52 06 AM" src="https://user-images.githubusercontent.com/9278015/88888747-86495b00-d25c-11ea-9a34-7304c2de5349.png">
<img width="1510" alt="Screenshot 2020-07-30 at 11 52 26 AM" src="https://user-images.githubusercontent.com/9278015/88888815-a1b46600-d25c-11ea-809c-4841c89dd21b.png">
<img width="1510" alt="Screenshot 2020-07-30 at 11 52 26 AM" src="https://user-images.githubusercontent.com/9278015/88888807-9eb97580-d25c-11ea-81c9-48829fa3535f.png">

Reply:
@sahil143 I just looked at PF font sizes. sm is in fact 14px - 
![image](https://user-images.githubusercontent.com/6041994/88908511-93c00e80-d277-11ea-994f-7f96d25a2b9d.png)